Dockerfile

```diff
@@ -1,12 +1,10 @@
-# hug depends on distutils which has been removed in python 3.12
-# so we are stuck with 3.11 for now
 FROM python:3.11-slim
 
 WORKDIR /dracor-metrics
 
 COPY . .
-RUN pip install poetry && poetry install
+RUN pip install poetry && poetry install --without dev
 
 EXPOSE 8030
 
-CMD [ "poetry", "run", "hug", "-f", "app/main.py", "-p", "8030" ]
```

Comment:
This was running the development server in the Docker image, right?

Reply:
This was running the service in the docker container.

Reply:
Yes, it this runs a development server nevertheless? I am not familiar with hug. But my understanding was, that most web frameworks run a local development server when invoked.
The recommendation for production mostly includes using another WSGI/ASGI server like Gunicorn or uwsgi.

Reply:
Yes, I think it is the same principle. It's running a dev server that is not really meant for production use.